src/components/latest-rules-content/latestRulesContent.js

```diff
@@ -129,6 +129,18 @@ const LatestRulesContent = ({
                     {item.item.frontmatter.title}
                   </Link>
                 </div>
+                {isShowAuthor && (
+                  <div className="hidden lg:block">
+                    <button
+                      onClick={() => openUserRule(item.file.node.file)}
+                      className="text-left cursor-pointer hover:text-ssw-red"
+                    >
+                      {item.file.node.lastUpdatedBy
```

Comment:
What will happen if there is no `lastUpdatedBy`?

I think we should handle this error

Reply:
> What will happen if there is no `lastUpdatedBy`?
> 
> I think we should handle this error

I've checked already, if no one updates the rule, the 'lastUpdatedBy' value will show the one who created the rule.
But I also update the code to avoid this potential error, thank you for your advice😊